### DISCUSSION TITLE
how to run in CPU float32 mode?

### DISCUSSION THREAD

**Discussion by iperov** [ggml-org/llama.cpp#9940] [Q&A]
I am running "Meta-Llama-3.1-8B-Instruct-f32.gguf" size is 32GB. 
RAM size is 64GB
After load llama-cli.exe uses 16GB according process explorer.

also in the log
```
llama_kv_cache_init:        CPU KV buffer size = 16384.00 MiB
llama_new_context_with_model: KV self size  = 16384.00 MiB, K (f16): 8192.00 MiB, V (f16): 8192.00 MiB
```

so how to run in float32 mode?

**Comment by ggerganov** [comment]
Add `-ctk f32 -ctv f32`

**Reply by iperov** [reply]
> Add `-ctk f32 -ctv f32`

got error
```
llama_new_context_with_model: V cache quantization requires flash_attn
```

but `-ctk f32 -ctv f16` works
